### DISCUSSION TITLE
Which team will win the UCL?

### DISCUSSION THREAD

**Discussion by c-gomez-m** [unaizorrilla-githubschool/AccionaNonDevelopers#18] [General]
# UCK Semifinals

## Real Madrid - Manchester City - semifinal 2

**Comment by GVilG** [comment]
Inter de Milán y Man. City

@c-gomez-m Real Madrid es un abusón y debería dejar ganar algo al resto

**Comment by c-gomez-m** [comment]
all details [here](https://es.uefa.com/uefachampionsleague/)

**Comment by jorubiop** [comment]
Inter
RM
